srcpkgs/gnome-characters/template

```diff
@@ -13,7 +13,8 @@ short_desc="Utility to find and insert unusual characters for GNOME"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Design/Apps/CharacterMap"
+changelog="https://gitlab.gnome.org/GNOME/gnome-characters/-/raw/main/NEWS"
```

Comment:
This should be `https://gitlab.gnome.org/GNOME/gnome-characters/-/raw/gnome-46/NEWS`